spec/factories/vm_azure.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :vm_azure, :class => "ManageIQ::Providers::Azure::CloudManager::Vm", :parent => :vm_cloud do
     location "westus"
-    vendor   "azure"
+    vendor   "microsoft"
```

Comment:
I saw the refresh code set it to Microsoft. https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb#L228